src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public static String getRandomColor() {
+        Color[] colors = Color.values();
+        Random random = new Random();
```

Comment:
move random to the class constant, like 
![image](https://user-images.githubusercontent.com/12864128/233040381-9698dd0b-a654-42b6-8af1-54224429fd72.png)